src/app/model/ProductIdQuantity.ts

```diff
@@ -0,0 +1,5 @@
+export interface ProductIdQuantity
```

Comment:
**suggestion:** try to follow the angular styling guide for naming files: https://angular.io/guide/styleguide#style-vocabulary.

In this case either name your file `product-id-quantity.ts` or you can just use the already defined type of Map<number, number>